lib/config.js

```diff
@@ -56,12 +57,11 @@ config.mongodb.writeOptions = {
     w: 'majority',
     j: true,
   },
-  forceServerObjectId: true,
 };
 
 config.mongodb.requirements = {};
 // server version requirement with server-style string
-config.mongodb.requirements.serverVersion = '>=4.2';
+config.mongodb.requirements.serverVersion = '>=4.4';
```

Comment:
Is 4.4 a requirement?  We do still have some clusters on Atlas that are 4.2.x.

Reply:
Ok so `4.4` is the recommended min version, you can set the serverVersion as low as 3.6. See the compatibility matrix here: https://www.mongodb.com/docs/drivers/node/current/compatibility/#mongodb-compatibility